amida-tech/indaba-client
# INBA-325

#### What's this PR do?
Fixes bug that occurs after creating a new project, that otherwise prevented you from returning to the new project wizard..

#### Related JIRA tickets:

#### How should this be manually tested?
Start the front and back end.
Go to `http://localhost:3000/login`
Login.
Click on "Create"
Click on "Create New Project"
Go through the steps and create a new project.
Once complete, click on "Go to Project"
Click on "Create"
Click on "Create New Project"
Confirm the "Completed Project" modal doesn't appear.

#### Any background context you want to provide?
#### Screenshots (if appropriate):
